pkg/stanza/operator/helper/encoding.go

```diff
@@ -19,10 +19,11 @@ import (
 	"fmt"
 	"strings"
 
+	"golang.org/x/text/transform"
+
 	"golang.org/x/text/encoding"
 	"golang.org/x/text/encoding/ianaindex"
 	"golang.org/x/text/encoding/unicode"
-	"golang.org/x/text/transform"
```

Comment:
This rearrangement of the imports will have to be reverted.

Reply:
fixed.